File: Atividade_6/Java/ex03.java
import java.util.Scanner;

public class ex03 {
    public static void main(String[] args) {
        System.out.println("Informe a quantidade de brinquedos vendidos: ");
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        final double precoBrinquedo = 19.9; 
        final double totalVendas = n * precoBrinquedo;

        int bonusQuant = n / 15;
        double valorBonus = bonusQuant * (totalVendas*0.08);
        double salario = (totalVendas/2) + valorBonus;

        System.out.println("Valor arrecadado com as vendas: R$" + totalVendas);
        System.out.println("Valor arrecadado com bônus: " + valorBonus);
        System.out.println("Salário total: " + salario);

        in.close();
    }
}

File: Atividade_6/Haskell/ex03.hs
-- Importando o módulo Text.Printf para formatar a saída
import Text.Printf (printf)

-- Função principal
main :: IO ()
main = do
    putStrLn "Informe a quantidade de brinquedos vendidos: "
    n <- readLn :: IO Int
    let precoBrinquedo = 19.9 :: Double
        totalVendas = fromIntegral n * precoBrinquedo
        bonusQuant = n `div` 15
        valorBonus = fromIntegral bonusQuant * (totalVendas * 0.08)
        salario = (totalVendas / 2) + valorBonus :: Double

    printf "Valor arrecadado com as vendas: R$%.2f\n" totalVendas
    printf "Valor arrecadado com bônus: R$%.2f\n" valorBonus
    printf "Salário total: R$%.2f\n" salario
